truss/templates/shared/logging.py

```diff
@@ -22,11 +22,37 @@ def filter(self, record: logging.LogRecord) -> bool:
         )
 
 
+class StreamToLogger:
+    """
+    StreamToLogger redirects stdout and stderr to logger
+    """
+
+    def __init__(self, logger, log_level, stream):
+        self.logger = logger
+        self.log_level = log_level
+        self.stream = stream
+
+    def __getattr__(self, name):
+        # we need to pass `isatty` from the stream for uvicorn
+        # this is a more general, less hacky fix
+        return getattr(self.stream, name)
+
+    def write(self, buf):
+        for line in buf.rstrip().splitlines():
```

Comment:
why do we do this line splitting thing? I think the intuitive thing as a user would be that 1 print = 1 logline, and it seems like we're trying to be a little clever by splitting the lines. wdyt?

Reply:
That's a good point. There's no real reason each line should be in a different log (just looked better from my end).